pkg/operator/collector-statefulset.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	// Buffer path for single process
+	// DefaultBufferPath Buffer path for single process
```

Comment:
```suggestion
	// Buffer path for single process
```

Reply:
Thanks for the PR, just wondering why adding function names like NewRoutePlugin, NewGlobalRouter, and DefaultBufferPath to the comments?

Reply:
Add variable descriptions before comments, which is a standard format requirement

Reply:
Using the function or parameter name as the beginning of the comment is correct, but we need to ensure that the comment is grammatical. Perhaps. 

`DefaultBufferPath Buffer path for single process` is not a good comment.
`DefaultBufferPath defines the buffer path for single process` is better.

I think these comments all need to be adjusted.

Reply:
updated